backend/apps/slack/utils.py

```diff
@@ -65,6 +83,48 @@ def get_news_data(limit=10, timeout=30):
     return items
 
 
+def get_or_create_issue(issue_link):
```

Comment:
This scope is to broad for /sponsor command's function. Let's split it and remove from here.